Text/CustomEntitySearch/CustomEntitySearch.cs

```diff
@@ -29,6 +29,9 @@ namespace AzureCognitiveSearch.PowerSkills.Text.CustomEntitySearch
     /// </summary>
     public static class CustomEntitySearch
     {
+        // Use this to load from "csv" or "json" file 
+        public static IList<string> preLoadedWords = new WordLinker("csv").Words;
```

Comment:
I'm fine with the default being CSV, but we should try to lookup an env variable first, and if present, use that. `WordLinker` could also check for file existence, and fallback gracefully.